src/zarr/storage/_fsspec.py

```diff
@@ -3,6 +3,8 @@
 import warnings
 from typing import TYPE_CHECKING, Any
 
+from fsspec.implementations.asyn_wrapper import AsyncFileSystemWrapper
```

Comment:
This makes the whole module depend on fsspec. I'm OK with that.

Reply:
Yeah, this is definitely unfortunate but a bit difficult to avoid

Reply:
why can't this import be done in or around line 162?

Reply:
Happy to do that (and add a bit of caution around the possibly failing import depending on fsspec versions)